admin/admin_styles.css

```diff
@@ -0,0 +1,58 @@
+.wrapper {
+    display: flex;
+    width: 100%;
+}
+
+#sidebar {
```

Comment:
Since we're using Bootstrap for the admin pages, can we go through and double check that all of this CSS is being used? And whatever is not being used, please remove

Reply:
Do the header and footer need a separate css file than the product form page?

Reply:
@shorticins I don't think we'll be doing a TON of custom CSS for the admin pages since we're using Bootstrap so I think one file is fine for now. If you do want to break them out I can create a task for you guys to setup a css directory using SASS

Reply:
cleaned up the css